docs/database/0001_init_podcasting_20_database.sql

```diff
@@ -0,0 +1,655 @@
+    is_resized BOOLEAN DEFAULT FALSE
+);
+
+CREATE TABLE channel_image (
+    channel_id INTEGER NOT NULL REFERENCES channel(id) ON DELETE CASCADE
+) INHERITS (image_base);
+
+CREATE TABLE chapter_image (
+    chapter_id INTEGER NOT NULL REFERENCES item_chapter(id) ON DELETE CASCADE
+) INHERITS (image_base);
+
+CREATE TABLE item_image (
+    item_id INTEGER NOT NULL REFERENCES item(id) ON DELETE CASCADE
+) INHERITS (image_base);
+
+--** LICENSE
+
+-- <podcast:license>
+CREATE TABLE license_base (
+    id SERIAL PRIMARY KEY,
+    type varchar_normal NOT NULL,
+    url varchar_url NOT NULL
+);
+
+CREATE TABLE channel_license (
+    channel_id INTEGER NOT NULL REFERENCES channel(id) ON DELETE CASCADE,
+    UNIQUE (channel_id)
+) INHERITS (license_base);
+
+CREATE TABLE item_license (
+    item_id INTEGER NOT NULL REFERENCES item(id) ON DELETE CASCADE,
+    UNIQUE (item_id)
+) INHERITS (license_base);
+
+--** LOCATION
+
+-- <podcast:location>
+CREATE TABLE location_base (
+    id SERIAL PRIMARY KEY,
+    geo varchar_normal,
+    osm varchar_normal,
+    CHECK (
+      (geo IS NOT NULL AND osm IS NULL) OR 
+      (geo IS NULL AND osm IS NOT NULL)
+    )
+);
+
+CREATE TABLE item_chapter_location (
+    item_chapter_id INTEGER NOT NULL REFERENCES item_chapter(id) ON DELETE CASCADE
+) INHERITS (location_base);
+
+CREATE TABLE channel_location (
+    channel_id INTEGER NOT NULL REFERENCES channel(id) ON DELETE CASCADE
+) INHERITS (location_base);
+
+CREATE TABLE item_location (
+    item_id INTEGER NOT NULL REFERENCES item(id) ON DELETE CASCADE
+) INHERITS (location_base);
+
+--** REMOTE ITEM
+
+-- <podcast:remoteItem>
+CREATE TABLE remote_item_base (
+    id SERIAL PRIMARY KEY,
+    feed_guid UUID NOT NULL,
+    feed_url varchar_url,
+    item_guid varchar_uri,
+    title varchar_normal
+);
+
+CREATE TABLE channel_remote_item (
+    channel_id INTEGER NOT NULL REFERENCES channel(id) ON DELETE CASCADE
+) INHERITS (remote_item_base);
+
+CREATE TABLE item_remote_item (
+    item_id INTEGER NOT NULL REFERENCES item(id) ON DELETE CASCADE
+) INHERITS (remote_item_base);
+
+CREATE TABLE podroll_remote_item (
+    podroll_id INTEGER NOT NULL REFERENCES channel_podroll(id) ON DELETE CASCADE
+) INHERITS (remote_item_base);
+
+--** MEDIUM
+
+-- <podcast:medium>
+CREATE TABLE medium_value (
+    id SERIAL PRIMARY KEY,
+    value TEXT UNIQUE CHECK (VALUE IN (
+        'publisher',
+        'podcast', 'music', 'video', 'film', 'audiobook', 'newsletter', 'blog', 'publisher', 'course',
+        'mixed', 'podcastL', 'musicL', 'videoL', 'filmL', 'audiobookL', 'newsletterL', 'blogL', 'publisherL', 'courseL'
+    ))
+);
+
+INSERT INTO medium_value (value) VALUES
+    ('publisher'),
+    ('podcast'), ('music'), ('video'), ('film'), ('audiobook'), ('newsletter'), ('blog'), ('course'),
+    ('mixed'), ('podcastL'), ('musicL'), ('videoL'), ('filmL'), ('audiobookL'), ('newsletterL'), ('blogL'), ('publisherL'), ('courseL')
+;
+
+CREATE TABLE medium_base (
```

Comment:
AH there we go-found the table :) Ignore my comment about missing the 'Medium' table